<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pButton_ImgList">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>图像文件</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>60</y>
      <width>131</width>
      <height>481</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skipButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>跳过</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doneButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>240</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>340</y>
      <width>95</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="rectButton">
       <property name="text">
        <string>矩形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AButton">
       <property name="text">
        <string>Point A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BButton">
       <property name="text">
        <string>Point B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CButton">
       <property name="text">
        <string>Point C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DButton">
       <property name="text">
        <string>Point D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EButton">
       <property name="text">
        <string>Point E</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1136</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
